src/encoded/static/scss/encoded/modules/_matrix.scss

```diff
@@ -621,6 +621,7 @@ $matrix-border-color: #f0f0f0;
             padding: 0;
             flex: 0 0 auto;
             height: 10px;
+            font-size: 0;
```

Comment:
This works. But it may be confusing as you are making  font-size = 0 and that moves the arrow. I would recommending modifying:

`matrix__category-expander svg`

and using something like 

`display:flex`